### ISSUE TITLE
[UI-FIX] ALL BUTTONS

### ISSUE THREAD

**Issue by moatazkhalifa** [asgiom/cr8-brm-mobile-issues#21]
Instead of creating a separate issue for each button, I'm going to list all button issues below.

- Buy/Sell buttons: Proceed to KYC & Recheck KYC Valid: text NEED NOT to be capitalized, button height and font is odd.
- Receive: Share Account: When the button is clicked, it goes all orange and become unreadable
- Send tab: Send CR8: Button has an odd color

Proposed fix:

Please create a class for ALL buttons and simply use it everywhere. You need not to style each button every time you add one, this becomes very hard to maintain and is a very problematic way to add buttons.

Below is an example class:

`.my-custom-button: {
    font-size: 16px;
    line-height: 16px;
    text-transform: none;
    text-decoration: none;
    background-color: #ff6500;
    border: 1px solid #ff6500;
    border-radius: 5px;
    padding: 12px 15px;
    font-weight: 500;
    letter-spacing: 0px;
    min-width: 100px;
    min-height: 40px;
}`

`.my-custom-button:hover {
    background-color: #000000;
    border: 1px solid #000000;
}`

Feel free to play with the code till it becomes identical to what we have on 'Receive' screen as that's the closest to what we need.

Then, use the same class for all buttons, everywhere.

**Comment by t3nshi** [comment]
We already had this styles for buttons app wide. Are you sure you want me to make the button black on hover? I can update some buttons with your provided style, the previous style was obtained from the pictures provided, but it was more round until you asked for a 5px border radius more squary look. we have like 3 type of buttons on the app. By the way, as I told you before, i could not reproduce the orange on orange thing before, but now I can. The issue is that no way to know what's producing it yet. I will let you know. 
the issue with Buy/sell is that it was created as an external component and it does not know of any of the app wide config, we have to pass it to it.... I will fix this soon but it is not a matter of just throwing a css. 
this are the current styles for the main buttons right now... they are app wide... 
``` css
ion-button.cr8-button{ /* this semi-transparent white button */
  --color: var(--ion-color-safety-orange); 
  --background: rgba(var(--ion-color-light-rgb), 0.7);
  text-transform: capitalize;
  --border-radius: 5px;
  // --border-radius: 26px;
  height: 52px;
  font-size:15px;
  font-weight: bolder;
  min-width: 52px;
}
ion-button.cr8.button:active,
ion-button.cr8.button:hover{
  --color: var(--ion-color-safety-orange);
}
ion-button.cr8-button-primary{ /* this solid orange button */
  --color: var(--ion-color-light);
  --background: rgba(var(--ion-color-primary-rgb), 1);
  text-transform: capitalize;
  // --border-radius: 26px;
  --border-radius: 5px;
  height: 52px;
  font-size:15px;
  font-weight: bolder;
  min-width: 52px;
}
ion-button.cr8.button-primary:active,
ion-button.cr8.button-primary:hover{
  --color: var(--ion-color-light); /* this was me trying to blindly fix the orange on orange */
}
ion-button.cr8-button-secondary{ /* this is the transparent with white text button */
  --color: var(--ion-color-light);
  --background: rgba(var(--ion-color-light-rgb), 0.4);
  text-transform: capitalize;
  // --border-radius: 26px;
  --border-radius: 5px;
  height: 52px;
  font-size:15px;
  font-weight: bolder;
  min-width: 52px;
}
```

**Comment by t3nshi** [comment]
> Send tab: Send CR8: Button has an odd color

of course... it should have a pale color all the time the button is disabled for some reason.

**Comment by t3nshi** [comment]
> Buy/Sell buttons: Proceed to KYC & Recheck KYC Valid: text NEED NOT to be capitalized, button height and font is odd.

I was able to include a temporary workaround while I finish implementing the passing of styles to the opibis component, so you can see the buttons how it should look even though they are not fully fixed.

**Comment by moatazkhalifa** [comment]
It's fine to keep the current hover color, we can re-visit that later. Not an issue.

My problem is that every button looks different on every tab. They should all look the same. 

Happy to use the same code on 'Receive' screen for all buttons. As long as they all look like that, It will be fine.

**Comment by t3nshi** [comment]
But they are almost like that.... except when the button is disabled, then it is a bit pale. And there are some buttons that are alike but white instead of orange... But if you are talking about the opibis component then that is a different story.

**Comment by moatazkhalifa** [comment]
For set limits, they're orange/white: ok
Settings: they're white/white: not ok
Receive > Share account: when tapped it goes all Orange, not ok
Buy/Sell: orange/orange and wrong dimensions, font

**Comment by t3nshi** [comment]
fixed in 0.4.7
